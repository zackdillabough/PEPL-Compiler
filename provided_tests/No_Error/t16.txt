POLY F4 = x + 1;
POLY F2 = x^2 + x + 4;
POLY F3(X) = X - 1;
START
INPUT X;
F4(X);
F2(10);
F3(F2(0));
F4(F3(F4(F3(F4(F3(F4(F3(F4(F3(F4(F3(F4(F3
(F4(F3(F4(F3(F4(F3(X))))))))))))))))))));
5 2 2 1 0
