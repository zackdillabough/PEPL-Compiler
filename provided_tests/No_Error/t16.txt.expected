6
114
3
5
